extensions/amp-video/0.1/amp-video.js

```diff
@@ -195,7 +200,11 @@ class AmpVideo extends AMP.BaseElement {
     };
 
     installVideoManagerForDoc(this.element);
-    Services.videoManagerForDoc(this.element).register(this);
+
+    // amp-story coordinates playback based on page activation, as opposed to
+    // visibility.
+    Services.videoManagerForDoc(this.element).register(this,
```

Comment:
I am okay with this temporarily but let's create an issue for a better solution. Normal autoplay behavior should work for `amp-story` even based on visibility of pages (same way it works for `amp-carousel`). It would require `amp-story` to manage `pages` using `schedule(layout,resume,pause)` similar to `amp-carousel`

Reply:
Filed #12712 to track.  Will follow up with some questions there.

Reply:
Thanks, added a TODO.